lambdaclass/curse_of_myrra
# Feedback for basic skill

Closes #810 

This PR implements a new feedback that is visualized on all those enemies that are in the range of a basic attack or Muflus' barrel roll or H4ck's Multishot

Steps to test:
1. Start a new game with at least two players (make sure to have at least one Muflus and one H4ck).
2. Use Muflus to get near to one of the other characters. Tap and hold the basic attack button, and notice how your enemy changes color when it's in your attack range.
3. Repeat for the barrel roll skill and the multishot skill.
4. Repeat for the basic attack of any other character.

Notes:
1. You currently won't see this feedback for D'Agna, as its basic attack is not yet configured for this purpose. But, if you want to verify that it will work when this is done, you can set a value for the `Skill Circle Radius` attribute in the `DAgna_Scherzo` scriptable object, and also set an Angle for this skill in the `skills.json` file.
2. For H4ck's case, the feedback will be shown only when the enemy is reachable in the SkillRange Area (that's the purple circle that is rendered at H4ck's feet when you tap and hold the basic attack button).
3. The current feedback we're using is temporary, and we'll make it nicer and more noticeable in a future PR.
